common/src/main/java/tachyon/security/PlainSaslHelper.java

```diff
@@ -67,7 +74,46 @@ public static TTransportFactory getPlainServerTransportFactory(AuthType authType
     return saslFactory;
   }
 
-  // TODO: get client side PLAIN TTransport
-//  public static TTransport getPlainClientTransport(String username, String password,
-//      TTransport wrappedTransport) throws SaslException {}
+  /**
+   * Get a ClientTransport for Plain
+   * @param username User Name of PlainClient
+   * @param password Password of PlainClient
+   * @param wrappedTransport The original Transport
+   * @return Wrapped Transport with Plain
+   * @throws SaslException if an AuthenticationProvider is not found
+   */
+  public static TTransport getPlainClientTransport(String username, String password,
+      TTransport wrappedTransport) throws SaslException {
+    return new TSaslClientTransport("PLAIN", null, null, null, new HashMap<String, String>(),
+        new PlainClientCallbackHandler(username, password), wrappedTransport);
+  }
+
+  public static class PlainClientCallbackHandler implements CallbackHandler {
```

Comment:
Fixed. I add a comment:  A client side callback to put application provided username/password into SASL transport.